pkgs/tools/misc/uhubctl/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv
+, fetchFromGitHub
+, libusb
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhubctl";
+  version = "2.0.0-dev-1961aa0";
```

```suggestion
  version = "unstable-2019-08-20";
```

As per https://nixos.org/nixpkgs/manual/#sec-package-naming

---

Fixed in fce13d627a249c0c4f27ff3f08a73beb9a02fabb